opentelemetry-api/src/trace/noop.rs

```diff
@@ -31,9 +31,9 @@ impl trace::TracerProvider for NoopTracerProvider {
     /// Returns a new `NoopTracer` instance.
     fn versioned_tracer(
         &self,
-        _name: impl Into<Cow<'static, str>>,
-        _version: Option<&'static str>,
-        _schema_url: Option<&'static str>,
+        _name: Cow<'static, str>,
```

Comment:
What are the implications of using `impl Into<Cow<'static, str>>` to `Cow<'static, str>`?